pkgs/by-name/ve/vegastrike/package.nix

```diff
@@ -0,0 +1,96 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, makeWrapper
+# python311 causes trouble
+# https://github.com/vegastrike/Vega-Strike-Engine-Source/issues/777
+# Will be fixed in next release. There is alternatively a patch that fixes this
+# and allows to switch to latest python if needed
+, python310
+, libpng
+, libjpeg
+, expat
+, gtk3
+, glib
+, openal
+, libogg
+, libvorbis
+, SDL
+, boost
+, python310Packages
+, freeglut
+, vegastrike-utcs-data
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vegastrike";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "vegastrike";
+    repo = "Vega-Strike-Engine-Source";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-8NwXp7PDh8DL777LZcpyC2UmKFwzg1XKIGrL5hzKpbg=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    makeWrapper
+  ];
```

Comment:
Do you need those two during the runtime? Otherwise I'd put them into `buildInputs`